dev-util/wsta/wsta-0.4.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="A CLI development tool for WebSocket APIs"
+HOMEPAGE="https://github.com/esphen/wsta/"
+SRC_URI="https://github.com/esphen/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="dev-libs/openssl:*"
```

Comment:
The package depends on [this library](https://github.com/sfackler/rust-openssl), which does not even mention libressl. I'll wager that it does not work, but I'll definitely test it out